extras/craft/gulprev/twigextensions/GulpRevTwigExtension.php

```diff
@@ -30,12 +30,13 @@ public function gulpRev($file)
 
         // If the file is not defined in the asset manifest
         // just return the original string
-        $path = $file;
+        $path          = $file;
+        $manifest_path = $_SERVER['DOCUMENT_ROOT'] . '/rev-manifest.json';
```

Comment:
Some servers can treat these vars differently -- what about `CRAFT_BASE_PATH . '/public/rev-menufest.json` ?  Does that give the correct path?